.github/workflows/code_analysis.yaml

```diff
@@ -62,4 +62,9 @@ jobs:
             # composer install cache - https://github.com/ramsey/composer-install
             -   uses: "ramsey/composer-install@v2"
 
+            # Override code from symplify/coding-standard shipped with ECS
+            -   run: |
+                    rm -rf vendor/symplify/easy-coding-standard/vendor/symplify/coding-standard/src
+                    ln -s $PWD/src vendor/symplify/easy-coding-standard/vendor/symplify/coding-standard/
```

Comment:
Another idea is to handle it with composer's [`post-install-cmd`](https://getcomposer.org/doc/articles/scripts.md) scripts instead of GitHub Actions. However, it is complicated by compatibility with Windows symlinks.

Reply:
For the reason of this comment https://github.com/symplify/coding-standard/pull/21#issuecomment-1528973676, tests always fail unless ECS is updated by merging this PR of symplify/coding-standard.